app/views/messages/index.html.erb

```diff
@@ -21,7 +21,7 @@
     <div class="grid__item width-two-thirds shift-one-sixth">
       <%= form_for @message, remote: true, :html => {class: 'sendbar form-width--searchbar' }, builder: GcfFormBuilder do |f| %>
           <%= f.text_field(:body, { autofocus: true, type: 'text', class: 'sendbar__input', autocomplete: 'off', autocorrect: 'off', autocapitalize: 'off', placeholder: 'Send a text message' }) %>
-          <button id="send_message" class="button button--primary" type="submit">Send</button>
+          <button id="send_message" class="sendbar__button button--primary" type="submit">Send</button>
```

Comment:
We're going to be changing the design of the send message form anyway (adding the `Send later` button) so lets not get too wrapped up in making the original design look right.